azure-pipelines.yml

```diff
@@ -14,254 +14,237 @@ trigger:
-      contents: '**'
-      targetFolder: '$(Build.ArtifactStagingDirectory)\teams-test-app'
-    displayName: 'Copy Test app to artifacts staging directory'
-
-  - task: ArchiveFiles@2
-    inputs:
-      rootFolderOrFile: '$(Build.ArtifactStagingDirectory)\teams-test-app'
-      includeRootFolder: false
-      archiveType: 'zip'
-      archiveFile: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
-      replaceExistingArchive: true
-    displayName: 'Zip Test app artifacts'
-
-  - task: PublishBuildArtifacts@1
-    inputs:
-      PathtoPublish: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
-      ArtifactName: 'teams-test-app'
-    displayName: 'Publish Test app artifacts'
-
-  # Retaining artifacts using Arrow service (https://aka.ms/m365sdlonboarding​)
-  - task: AssetRetention@3
-    inputs:
-      ArrowServiceConnection: '$(ArrowConnection)'
-      AssetGroupName: '$(System.TeamProject)_$(Build.DefinitionName)'
-      AssetNumber: '$(Build.BuildId)'
-      IsShipped: true # This setting will ensure the artifacts are retained more than 548 days as per compliance, more details https://microsoft.sharepoint.com/teams/M365Security/EngSec/SitePages/How-to-make-sure-your-SDL-results-are-retained.aspx
-      DropsToRetain: 'CodeAnalysisLogs'
-    condition: and(
-      startsWith(variables['Build.SourceBranch'], 'refs/tags/v'),
-      in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
-      ne(variables['Build.Reason'], 'PullRequest'),
-      ne(variables['Build.Reason'], 'Manual')
-      )
-    displayName: 'Artifact Retention(Arrow)'
-
-  - task: PostAnalysis@2
-    condition: succeededOrFailed()
-    displayName: 'Guardian Break'
-    inputs:
-      GdnBreakPolicyMinSev: Warning
-      GdnBreakAllTools: true
-      GdnBreakGdnToolCredScan: true
-      GdnBreakGdnToolCredScanSeverity: Warning
-      GdnBreakGdnToolESLint: true
-      GdnBreakGdnToolESLintSeverity: Warning
-      GdnBreakPolicy: M365
-
-  - task: AzureRmWebAppDeployment@4
-    condition: and(
-      in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
-      startsWith(variables['Build.SourceBranch'], 'refs/tags/v'))
-    inputs:
-      ConnectionType: 'AzureRM'
-      azureSubscription: '$(AzureSubscription)'
-      appType: 'webApp'
-      WebAppName: 'musicalsink'
-      package: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
-    displayName: 'Deploy to musicalsink'
-
-  - task: AzureRmWebAppDeployment@4
-    condition: and(
-      in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
-      eq(variables['Build.SourceBranch'], 'refs/heads/2.0-preview'))
-    inputs:
-      ConnectionType: 'AzureRM'
-      azureSubscription: '$(AzureSubscription)'
-      appType: 'webApp'
-      WebAppName: 'cloudroll'
-      package: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
-    displayName: 'Deploy to cloudroll'
```

Comment:
These tasks don't appear in the resulting changes. To confirm - these are being done by the release pipeline now?

Reply:
Confirmed offline that they are.

Reply:
Correct